### DISCUSSION TITLE
Easy linking between documents

### DISCUSSION THREAD

**Discussion by EacAyf** [outline/outline#2490] [Ideas]
Just started using Outline and I'm loving it. Aside from some other great idea's I've seen here I would love it if there would be an easy way of linking documents to one and another. A lot of the documentation we have are interlinked and it would be great to have an option in the "/" menu to link towards a certain document/heading within that document.  

I'd like to hear your thoughts on this.

**Comment by tommoor** [comment]
Hey, did you see `/link` in the menu? You can also do `CMD+K`, type the name of another document and then hit enter to link it up fast. Let me know if this does the trick

**Reply by EacAyf** [reply]
Ok, wow, I have no idea how I was able to miss that now tha you pointed it out.
Thanks!
